### DISCUSSION TITLE
Pettier conflict with package.json

### DISCUSSION THREAD

**Discussion by VPKSoft** [remirror/remirror#2233] [Q&A] [answered]
Hi,
I don't know if this is a bug or feature so I'll just as the question first.
I Tried to install **remirror** with the following command on my project:
`npm install remirror @remirror/react @remirror/pm` 

However I got prettier conflict with *@remirror/extension-code-block*:
```
npm WARN ERESOLVE overriding peer dependency
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: remirror@2.0.38
npm ERR! Found: prettier@3.2.2
npm ERR! node_modules/prettier
npm ERR!   peerOptional prettier@"^2.8.8" from @remirror/extension-code-block@2.0.18
npm ERR!   node_modules/remirror/node_modules/@remirror/extension-code-block
npm ERR!     @remirror/extension-code-block@"^2.0.18" from remirror@2.0.38
npm ERR!     node_modules/remirror
npm ERR!       remirror@"*" from the root project
...
```

Might there be an issue that prettier is installed in the `dependencies` instead of `devDependencies` with the `@remirror/extension-code-block@2.0.18`  extension?

**Comment by jvass-520** [comment, marked as answer]
@VPKSoft 

I think your issue is that you have two different prettier versions installed and they are conflicting.

You can try one of the following:
1. Install the code block extensions without the peer dependencies: `npm install @remirror/extension-code-block --no-optional`
2. Install the same prettier version `2.0.18` instead of the latest

Hope that helps.
